src/webhook/handler/main.ts

```diff
@@ -0,0 +1,101 @@
+import type { Bot, Handler, Interaction } from "../../../deps.ts";
```

Comment:
I wrote these codes with reference of 👇🏼 
https://github.com/discordeno/serverless-deno-deploy-template/blob/main/mod.ts

So, it may work properly... 🙏🏼